tests/cli/test_suite.py

```diff
@@ -3288,7 +3532,7 @@ def suite_new_messages():
             None,
             False,
             None,
-            True,
+            CLISuiteInteractiveFlagCombinations.UNPROMPTED_INTERACTIVE_TRUE_MANUAL_FALSE,
```

Comment:
Looks like we need a new message type `CLISuiteInteractiveFlagCombinations.UNPROMPTED_INTERACTIVE_TRUE_MANUAL_FALSE_PROFILE_TRUE` to capture that this user is using the profile flag. See https://superconductive.atlassian.net/browse/GREAT-434